application/modules/admin/views/scripts/shipment/ship-it.phtml

```diff
@@ -17,8 +17,14 @@
             ?>
                     <li><i class="fa fa-minus"></i>
                         <label>
-                            <input id="node-<?php echo $ckey; ?>" data-id="custom-<?php echo $ckey; ?>" type="checkbox"  <?php if(count($unenrolledParticipants) == 0) {  ?> checked="true" <?php  } ?>>
+                        <input id="node-<?php echo $ckey; ?>" data-id="custom-<?php echo $ckey; ?>" type="checkbox"  <?php if(count($unenrolledParticipants) == 0) {  ?> checked="true" <?php  } ?>>
                             <?php echo $ckey; ?>
+                        <input type="text" name="country_last_dates[<?php  echo $ckey; ?>]" size="50" maxlength="45" 
+                            value="<?php  echo $this->shipment->lastdate_response;?>"
```

Comment:
This should only be populated with the shipments lastdate_response when there isn't already a value specified for for the country. If there is a value saved for the country, this input box must show the date that has been saved for that country.